project/Dependencies.scala

```diff
@@ -77,6 +77,7 @@ object Dependencies {
   val scanamo = "com.gu" %% "scanamo" % "0.8.3"
   val enumeratumPlayJson = "com.beachape" %% "enumeratum-play-json" % "1.5.6"
   val commercialShared = "com.gu" %% "commercial-shared" % "6.0.0"
+  val jenkins = "com.gu" %% "jenkins" % "0.1-SNAPSHOT"
```

Comment:
I know naming is hard but I personally prefer name that give an hint of what the library is doing. Also jenkins is a very popular CI tool. This is a bit confusing.